WTraining/app/serializers/weet.js

```diff
@@ -0,0 +1,13 @@
+exports.allWeetsSerializer = ({ rows, count, qty }) => ({
+  weets: rows.map(weet => this.weetSerializer(weet)),
```

Comment:
When the service return a list of objects and pagination usually the object contains a rows key with this list  and pagination data is the same label for example:

`
{
     rows: rows.map(weet => this.weetSerializer(weet)),
     total: count
     total_page: xxx
     total_count: xxxx
     previous_page: xxx
     current_page: xxxx
     next_page: xxxx
}
`